apps.json

```
@@ -2094,7 +2099,7 @@
     },
     "kimai2": {
         "category": "productivity_and_management",
-        "level": 8,
+        "level": 0,
```

```suggestion
        "level": 6,
```

as per https://ci-apps.yunohost.org/ci/job/12512
Upgrades tests will most likely fail, to be fixed in the repo.